config/args.js

```diff
@@ -9,6 +9,12 @@ const optionDefinitions = [
     alias: 'e',
     type: String,
     defaultValue: 'production'
+  },
+  {
+    name: 'tenant',
+    alias: 't',
+    type: String,
+    defaultValue: 'seek'
```

Comment:
I'm not sure this should be provided as a default. Maybe if we choose a default, it should be the first tenant defined in your sku config or something?